requirements/base.txt

```diff
@@ -17,6 +17,7 @@ pip-tools==5.4.0          # via -r requirements/base.in
 plotly==4.14.1            # via -r requirements/base.in
 python-dateutil==2.8.1    # via pendulum
 python-rex==0.4           # via -r requirements/base.in
+pytz==2020.4              # via -r requirements/base.in
```

Comment:
Whats the background of injecting this dependency?

Reply:
https://github.com/gridsingularity/d3a/pull/1035#issue-562341305

Reply:
Okay, but you have added it as dependency without using it anywhere in code

Reply:
It is used internally in pickle, when deserialising timezone aware timestamps. Pickle is (also) used when sending jobs via RQ.

Reply:
Okay, I guess it could be removed from [pandapower](https://github.com/gridsingularity/d3a/blob/master/requirements/pandapower.txt#L22) dependency then as if powerflow is executed it would be executed on top of base dependencies

Reply:
To provide some more feedback, it is used not only in pickle, but in Python in general https://docs.python.org/3/library/zoneinfo.html#module-zoneinfo. In case Python fails to detect the timezone in a system, it will try to find this package (pytz) in order to set the correct timezone for the system. You can check also the description of the package here https://github.com/python/tzdata . 
I would recommend to add it even as a failsafe, despite the fact that it is not used by our code (it will be used internally by Python if there is no timezone in a system). Keep in mind that this is a first-party package, therefore it will probably be quite painless to keep it in our requirements.

Reply:
@faizan2590 pytz must be a requirement of pandapower itself, because `pip-compile requirements/pandapower.in` adds it to `pandapower.txt` automatically. But now I upgraded pytz from 2020.4 to 2020.5

Reply:
@hannesdiedrich I think this could be avoided via adding 
`-r base.txt`
 at the top of `pandapower.txt`

Reply:
@faizan2590 but then I have all the dependencies of base.txt also in pandapower.txt

Reply:
I think thats okay as pandower dependency would work with d3a on top of base dependencies BUT lets leave it for now as we are not using it ATM.